api/control/v1alpha1/types.go

```diff
@@ -128,28 +119,52 @@ const (
 
 // DownsyncPolicyClause identifies some objects (by a predicate)
 // and modulates how they are downsynced.
-// One modulation is specifying a set of StatusCollectors to apply
-// to returned status.
-// The other modulation is specifying whether the propagation from WDS to WEC
-// involves continual maintenance of the spec or the object is only created
-// if it is absent.
 type DownsyncPolicyClause struct {
 	DownsyncObjectTest `json:",inline"`
+	DownsyncModulation `json:",inline"`
+}
 
+// DownsyncModulation is about variations on downsync behavior.
+type DownsyncModulation struct {
 	// `createOnly` indicates that in a given WEC, the object is not to be updated
 	// if it already exists.
 	// +optional
 	CreateOnly bool `json:"createOnly,omitempty"`
 
-	// `statusCollection` holds the rules for collecting status from the WECs.
+	// `statusCollectors` is a list of references of StatusCollectors to apply.
 	// +optional
-	StatusCollection *StatusCollection `json:"statusCollection,omitempty"`
-}
-
-// StatusCollection holds the rules for collecting status from the WECs.
-type StatusCollection struct {
-	// `statusCollectors` is a list of StatusCollectors name references to be applied.
 	StatusCollectors []string `json:"statusCollectors,omitempty"`
+
+	// WantSingletonReportedState, in short, indicates an expectation
+	// that the matching workload objects are distributed to exactly one WEC
+	// and requests that the `.status` of such objects propagate from the WEC
+	// to the WDS.
+	//
+	// For a precise description, start with a couple of definitions.
+	// For a given workload object, _singleton status return is requested_
+	// if and only if there exists at least one BindingPolicy or Binding
```

Comment:
Does "at least one BindingPolicy or Binding" imply that Bindings could be made separately?

Reply:
It does not imply that. It is worded that way in order to be a useful definition regardless of whether or not there can be a Binding that is not derived from a BindingPolicy.